mp-backend/db-tables.yml

```diff
@@ -22,6 +22,33 @@ authTable:
       ProvisionedThroughput:
           ReadCapacityUnits: 1
           WriteCapacityUnits: 1
+    ProvisionedThroughput:
+      ReadCapacityUnits: 1
+      WriteCapacityUnits: 1
+recipeTable:
+  Type: AWS::DynamoDB::Table
+  Properties:
+    TableName: recipe
+    AttributeDefinitions:
+      - AttributeName: id
+        AttributeType: S
+    KeySchema:
+      - AttributeName: id
```

Comment:
Maybe we should be a little more descriptive with our names here, just because there are going to be a lot of IDs from different tables. Call this a recipeId. In addition each recipe will have a userID as well.